core/cap-0059.md

```diff
@@ -21,7 +21,9 @@ BLS12-381 is a pairing-friendly elliptic curve which enables a new suite of appl
 As described in the preamble section.
 
 ## Motivation
-Pairing friendly elliptic curve operations are the backbone of many advanced Zero Knowledge (ZK) constructions such as as zkSNARKs, zkSTARKs, PLONK, BLS Signatures and more. These, in turn, enable a wide array of applications ranging from scaling, to identity management and privacy. BLS12-381 is one of the most well-known and widely adopted pairing friendly curves, due to its efficiency and 128-bit security. However, the curve operations are inherently computationally intensive, making them prohibitive to be implemented in a smart contract. Providing an efficient implementation for a well-chosen set of curve and field operations natively inside the Soroban host is crucial for unlocking privacy-preserving applications in smart contracts.
+Pairing friendly elliptic curve operations are the backbone of many advanced Zero Knowledge (ZK) constructions, with wide array of applications ranging from scaling, to identity management and privacy preserving solutions. They are crucial for unlocking zero knowledge functionality in smart contracts.
```

Comment:
nit: with a wide array

Reply:
Also, adding ZK-login and BBS signature as examples in the motivation might help.

Reply:
fixed the wording. 
I don't think mentioning specific use case is useful, since they are not representative of the wide spectrum of applications out there.

Reply:
It was a suggestion from Legal actually. They thought since we say `privacy preserving solutions` providing some examples might make it better.